tests/security/MemberTest.yml

```diff
@@ -42,7 +42,7 @@ Member:
   test:
     FirstName: Test
     Surname: User
-    Email: sam@silverstripe.com
+    Email: dummy@silverstripe.tld
```

Comment:
I'd prefer using `@example.com` which will never become a real domain rather than one that just isn't real yet.